assets/src/blocks/CarouselHeader/CarouselHeaderEditor.js

```diff
@@ -124,8 +124,8 @@ export const CarouselHeaderEditor = ({setAttributes, attributes}) => {
             </Slide>
           ))}
           <CarouselControls
-            goToPrevSlide={goToPrevSlide}
-            goToNextSlide={goToNextSlide}
+            goToPrevSlide={() => goToPrevSlide(carousel_autoplay)}
```

Comment:
Previously it never received the `autoplay` value. It had been received the `event` as a parameter